.github/workflows/release.yml

```
@@ -60,6 +60,7 @@ jobs:
       - name: Upload documentation to Hackage
         run: |
           cabal upload --publish \
-            --token "${{ secrets.HACKAGE_API_KEY }}"
+            --username "${{ secrets.HACKAGE_USERNAME }}" \
+            --password "${{ secrets.HACKAGE_PASSWORD }}" \
```

Passing a username and password for authentication is very unsafe. It's better if we use the `Authorization` header to pass the API key instead:
```suggestion
            -H "Authorization: X-ApiKey ${{ secrets.HACKAGE_API_KEY }}" \
```